Add <Azure Container Apps - ARM template> deployment (#313)
1Password/scim-examples: azure-container-apps-arm/README.md

   - **Subscription** : Choose the subscription you prefer.
   - **Resource group**: Choose an existing Resource Group or create a new one using **Create new** button.
   - **Region**: Choose the region you prefer.
   - **Container App Name**: Enter a name you'd like to use, default will be `op-scim-con-app`.
   - **Container App Env Name**: Enter a name you'd like to use, default will be `op-scim-con-app-env`.
   - **Container App Log Analytics Name**: Enter a name you'd like to use, default will be `op-scim-con-app-env`.
   - **Scimsession** : Paste the entire contents of the `scimession` file.
   - **Workspace Actor** *(Only if you are using Google Workspace as your identity provider)*: Enter the email address of a Google Workspace administrator for the service account. If you are not using Google Workspace as your identity provider, leave this field blank.
   - **Workspace Credentials** *(Only if you are using Google Workspace as your identity provider)*: Paste the entire contents of the json key file you downloaded in the [before you begin](#before-you-begin) section. If you are not using Google Workspace as your identity provider, leave this field blank.
> [!IMPORTANT]

> > If you are not using Google Workspace as your identity provider, leave `Workspace Actor` and `Workspace Credential` fields blank.

Reviewer: I suggest saying this in a slightly different way:

```suggestion
> The optional `Workspace Actor` and `Workspace Credentials` fields should only be filled if integrating with Google Workspace for automated provisioning.
```

3. Click **Review + create**.

> 4. Once the validation succeeds, click **Create**. It will take a couple of minutes to run the deployment.

Reviewer: Nitpick:

```suggestion
4. Once the validation succeeds, click **Create**. It is expected to take a couple of minutes to complete the deployment.
```

## Step 3: Test your SCIM bridge

> Once your deployment is complete, click **Go to resource group** and click on the Container App you created.

Reviewer: Case (Container Apps is the product name, but MS uses lower case for each individual instance in their docs, so we should do the same):

```suggestion
Once your deployment is complete, click **Go to resource group** and click on the container app you created.
```

<hr>

## Update your SCIM bridge in the Azure Portal

> [!TIP]
> Check for 1Password SCIM Bridge updates on the [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).

1. Within your deployed 1Password SCIM Bridge Container App in the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Containers** from the sidebar.
2. Click **Edit and deploy**.
3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
4. Change the version number **2.9.5** in the **Image and Tag** field, **1password/scim:v2.9.5** to match the latest version from our [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).
5. Select **Save**.
6. Select **Create** to deploy a new revision using the updated image.
7. Enter your SCIM bridge URL in a browser and sign in with your bearer token.
8. Check the top left-hand side of the page to verify you're running the updated version of the SCIM bridge.

> After you sign in to your SCIM bridge, the [Automated User Provisioning page](https://start.1password.com/integrations/active/) in your 1Password account will also update with the latest access time and SCIM bridge version.

Reviewer: We should maybe consider this in a new PR, but I feel like applying the template against the existing deployment should be the way we instruct folks to update this.

> 
> 

Reviewer: Extra line here for some reason:

```suggestion

```

To use a new `scimsession` credentials file for your SCIM bridge, replace the secret in your Container App:

<details>
<summary>Replace your <code>scimsession</code> secret using the Azure Portal</summary>

1. Open the Azure Portal and go to the [Container Apps](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps) page.
2. Choose **Secrets** from the Settings section in the sidebar.
3. Edit the **scimsession** secret and paste the entire contents of your new `scimsession` file.
4. Select the checkbox and click **Save**.
5. Choose the **Revisions** from the Application section in the sidebar.
6. Click your current active revision and choose **Restart** in the details pane.
7. Enter your SCIM bridge URL in another browser tab or window and sign in using your new bearer token to [test your SCIM bridge](#step-5-test-your-scim-bridge).
8. Update your identity provider configuration with the new bearer token.
</details>

<details>
<summary>Replace your <code>scimsession</code> secret using the Azure Cloud Shell or AZ CLI</summary>

Using the Azure Cloud Shell or AZ CLI, follow the steps in our [Advanced guide](ADVANCED.md).

> </details>

Reviewer: Same as update instructions. Can we use the template itself to update creds?
